src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/Makelangelo5.java

```diff
@@ -2,18 +2,13 @@
 
 import com.jogamp.opengl.GL2;
 import com.jogamp.opengl.util.texture.Texture;
-import com.jogamp.opengl.util.texture.TextureIO;
-import com.marginallyclever.convenience.FileAccess;
 import com.marginallyclever.convenience.Point2D;
 import com.marginallyclever.makelangelo.plotter.Plotter;
 import com.marginallyclever.makelangelo.plotter.settings.PlotterSettings;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
+import static com.marginallyclever.makelangelo.plotter.plotterRenderer.DrawingAssistant.*;
```

Comment:
I'm dumb.  why 'static'?

Reply:
`import static` enables you to call method of `DrawingAssistant` like they were defined in your class or a parent.
`Makelangelo5.texture1 = loadTexture("/textures/makelangelo5.png");` <= note here that the call to`loadTexture()` is not `DrawingAssistant.loadTexture()` but just `loadTexture()`
Another example is in unit test where methods like `assertEquals() came from an `import static org.junit.jupiter.api.Assertions.assertEquals;`